specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/bastionHost.json

```diff
@@ -290,6 +262,249 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/bastionHosts/{bastionHostName}/createbsl": {
```

Comment:
>createbsl [](start = 131, length = 9)

consider  /createSharableLink  (baston is implicit)

Reply:
probably plural /createSharableLinks

---
In reply to: [367156955](https://github.com/Azure/azure-rest-api-specs/pull/7969#discussion_r367156955) [](ancestors = 367156955)

Reply:
Updated